CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
 # Changelog
+## [1.3.0] - 2021-03-24
+### Changes
+- Use inclusive language
+  - "slave" is deprecated in favor of "port"
+  - "master" is deprecated in favor of "controller"
```

Comment:
Please use backticks for the technical terms instead of double quotes.

Reply:
I replaced double quotes with the backticks.